### PR TITLE
Containerization for Bot and Linter Integration

### PR DESCRIPTION
**Pull request by mike-rus**
Added container for bot and development, and added python formatting linter

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+BOT_IMAGE = bot_image
+
+build_dev_image:
+	$(DOCKER) build -t $(DEV_IMAGE) . -f Dockerfile.dev
+
+run_dev_container: build_dev_image
+	$(DOCKER) run -it -e USER_ID=1000 -e GROUP_ID=1000 -v $(PWD):/home/whore $(DEV_IMAGE)
+
+lint_python-formatting: build_dev_image
+	$(DOCKER) run -it --rm -e USER_ID=1000 -e GROUP_ID=1000 -v $(PWD):/home/whore $(DEV_IMAGE) ./scripts/python-linter-formatting.sh
```
Comment (polina-gritsenko) [inline Makefile]: @mike-rus did not manage to get `-v $(PWD):/home/whore $(DEV_IMAGE) ` working on windows
changing `$(PWD)` to `"$(CURDIR)"` works
the whole command syntax:
```
lint_python-formatting: build_dev_image
	$(DOCKER) run -it --rm -e USER_ID=1000 -e GROUP_ID=1000 -v "$(CURDIR)":/home/whore $(DEV_IMAGE) ./scripts/python-linter-formatting.sh
```

Comment (mike-rus) [reply Makefile]: Yes, that's correct, this variable would be better suited for cross-platform usage

Comment (mike-rus) [reply Makefile]: fixed

**Review by polina-gritsenko — COMMENTED** [review]
bravo)

**Review by polina-gritsenko — APPROVED** [review]

**Pull request merged**